util/dvsim/qsubopts.py

```diff
@@ -1818,19 +1818,31 @@ def execute(self, mode='local', path=''):
 
         args = getattr(self.args, 'command_args', [])
         args = getattr(self.args, 'xterm_args', args)
-
-        exestring = ' '.join([program] + options + [self.args.command] + args)
+        # ---------------- command file -------------
+        cwd = os.getcwd()
+        command_file = cwd + '/command_file_' + str(os.getpid())
+        try:
+            with open(command_file, 'w') as f_command:
+                command_temp = str(self.args.command)
+                command_temp = command_temp.replace('"', '')
+                f_command.write(str(command_temp))
```

Comment:
Why the extra "str" here?

Reply:
Thanks for the review it is indeed redundant. Fixed.